packages/statemanager/src/index.ts

```diff
@@ -1,4 +1,5 @@
-export { StateManager, EIP2929StateManager } from './interface'
+export { StateManager, AccountFields, StateAccess } from './interface'
 export { BaseStateManager } from './baseStateManager'
 export { default as DefaultStateManager, Proof } from './stateManager'
 export { default as TransientStorage } from './transientStorage'
```

Comment:
I have a strong tendency to keep all this `TransientStorage` stuff in the VM and just do not expose anything anywhere, since EIP-1153 is still a very early-on experimental implementation.

I wouldn't want to already bake this so deep into our code base, so maybe just put pack somewhere in `evm`.

Reply:
Update: ah, just seeing that this is used in `EEI` (which I would want to get out of the `EVM` package along the work on #1831), so I am getting a bit unsure here how to handle and where to place.

Reply:
I can just move it back there, so a separate PR can take care of it, can't see `TransientStorage` linked anywhere in `StateManager`